src/main/java/io/vertx/core/http/impl/HttpUtils.java

```diff
@@ -126,6 +126,20 @@ public static String normalizePath(String pathname) {
       return "/";
     }
 
+    boolean needsChange = false;
```

Comment:
`boolean needsChange = pathname.charAt(0) != '/'`

Reply:
I think this way is clearer when you come back and read it as to exactly what is going on. I also don't think it makes any real difference either way.